lib/csv_helper/loader.rb

```diff
@@ -18,20 +24,34 @@ def load(file, options = {})
 
     private
 
-    def load_from_csv(file, options)
-      records = { valid: [], invalid: [] }
-
+    def load_from_csv(file, records, options)
       # Since row indexes start at 0 and spreadsheets on line 1,
       # add 1 for the difference in indexes and 1 for the header row itself.
       row_offset = 2 + (options[:skip_lines] || 0)
       SmarterCSV.process(file, merge_options(options)).each.with_index do |row, i|
         record = row_to_record(row, i + row_offset)
-        record.errors.any? ? records[:invalid] << record : records[:valid] << record
+        save_record_to_records(records, record)
+      end
+
+      records
+    end
+
+    def load_from_csv_with_version(file, records, options)
+      version = Version.preview_version
+      row_offset = 2 + (options[:skip_lines] || 0)
```

Comment:
2 appears to be a magic number, can we specify a constant and use the constant here?